README.md

```diff
@@ -0,0 +1,383 @@
+pypinfo: View PyPI download statistics with ease.
+=================================================
+
+[![version](https://img.shields.io/pypi/v/pypinfo.svg?style=flat-square)](https://pypi.org/project/pypinfo)
+[![python](https://img.shields.io/pypi/pyversions/pypinfo.svg?style=flat-square)](https://pypi.org/project/pypinfo)
+[![license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://en.wikipedia.org/wiki/MIT_License)
+[![license](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat-square)](https://github.com/psf/black)
+
+pypinfo is a simple CLI to access `PyPI`_ download statistics via Google's BigQuery.
+
+## Table of contents
+
+1. [Usage](#usage)
+2. [How to run](#installation)
+3. [Credits](#credits)
+
+## Usage
+
+<details>
```

Comment:
Shall we also add the installation instructions into a `<details>`? The screenshots are big and you only really need to see them once.

Reply:
Well thought. I had this sensation as well. Another option is to collapse only the picture.

Reply:
I've done the latter: https://github.com/brunolnetto/pypinfo/blob/master/README.md